src/components/pages/Profile/EditProfile.js

```diff
@@ -35,10 +35,36 @@ function EditProfile({ userInfo }) {
   const buttonStyle = {
     backgroundColor: '#003D71',
     color: '#ffffff',
+    alignItems: 'center',
+    backgroundImage:
+      'radial-gradient(100% 100% at 100% 0, #5adaff 0, #5468ff 100%)',
+    appearance: 'none',
+    border: 0,
+    borderRadius: '6px',
+    boxShadow:
+      'rgba(45, 35, 66, .4) 0 2px 4px,rgba(45, 35, 66, .3) 0 7px 13px -3px,rgba(58, 65, 111, .5) 0 -3px 0 inset',
+    boxSizing: 'border-box',
+    //   color: #fff;
```

Comment:
This commented out code should probably be removed

Reply:
yes. That was my mistake. I realized it while shooting the video.
i will remove it on my next correction.